NEWS.md

```diff
@@ -10,6 +34,10 @@
 relevant to determine the forecast unit. This can lead to unexpected behaviour, so setting the forecast unit explicitly can help make the code easier to debug and easier to read (see issue #268). 
 When used as part of a workflow, `set_forecast_unit()` can be directly piped into `check_forecasts()` to
 check everything is in order.
+- Made `check_forecasts()` and `score()` pipeable (see issue #290). This means that
```

Comment:
didn't these things happen in this PR? How can they be in a different package version>

Reply:
aah yes. That's me being stupid...